octobot_trading/personal_data/positions/position.py

```diff
@@ -188,7 +188,7 @@ def _update(self, position_id, symbol, currency, market, timestamp,
 
         self._update_quantity_or_size_if_necessary()
         # update side after quantity as it relies on self.quantity
-        self._update_side()
+        self._update_side(not entry_price)
```

Comment:
For now we reset the entry_price every time :open_mouth:

Reply:
when the side is changing yes (only happening at the 1st position init)